src/index.js

```diff
@@ -158,9 +158,10 @@ function getTestMetadataAssignment(state, t) {
  * @returns Babel variable declaration
  */
 function getTestMetadataDeclaration(state, t) {
+  const contextIdentifier = state.opts.contextIdentifier;
   const getTestMetadataExpression = t.callExpression(
     t.identifier(state.opts.getTestMetadataUID.name),
-    [t.thisExpression()]
+    [contextIdentifier]
```

Comment:
is it possible to us `t.identifier('QUnit.config.current.testEnvironment')` here, and remove part of the diff below? that way we wouldn't need to store more properties on `state`